app/src/main/res/values/styles_fonts.xml

```diff
@@ -94,16 +95,21 @@
         <item name="android:textSize">12sp</item>
     </style>
 
-    <style name="WalletTheme.TextAppearance.Medium16">
-        <item name="android:textColor">@color/textIconPrimary</item>
-        <item name="fontFamily">@font/manrope_medium</item>
+    <style name="WalletTheme.TextAppearance.Medium20" parent="WalletTheme.TextAppearance.Medium">
+        <item name="android:textSize">20sp</item>
```

Comment:
Why did you remove font here and below?

Reply:
i moved it to parent -> WalletTheme.TextAppearance.Medium

Reply:
oh okay